images.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Image struct {
+	Name string
+	Tags []Tag
+}
+
+type Tag struct {
+	Sha string
+	Tag string
+}
+
+var Images = []Image{
+	Image{
+		Name: "quay.io/coreos/hyperkube",
+		Tags: []Tag{
+			Tag{
+				Sha: "5ff22b5c65d5b93aa948b79028dc136a22cda2f049283103f10bd45650b47312",
+				Tag: "v1.5.2_coreos.2",
+			},
+		},
+	},
+}
```

Comment:
I am wondering if this is the right place for this configuration. I see this as configuration for a G8S installation like we already have in architect. What do you think in terms of future directions? Can/should we consolidate the image table with the rest of the settings a G8S installation needs? I can see this getting out of hand when it is in different places. How will the workflow look like when we have different installations using different registry endpoints? E.g. a customer probably wants to have its own registry and we need to sync our images with this one instead of quay. Thinking loud here.